### ISSUE TITLE
about the IRHeader when using pack and unpack

### ISSUE THREAD

**Issue by HaoLiuHust** [apache/incubator-mxnet#14499]
#12544 I am packing a multi-label dataset, and use header.flag, but after pack and unpack, the flag will be always equal to the label length, is it by design or a bug?

**Comment by zachgk** [comment]
I am looking at the code for IRHeader, pack, and unpack and it looks like it is by design https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/recordio.py#L340. The flag is used for label size throughout. If so, we should update the documentation to better explain this.

Any idea about this @Roshrini?

@mxnet-label-bot add [Data-loading]

**Comment by HaoLiuHust** [comment]
yes, I have read it too, if we want to use it as we want, we can write a custom pack and unpack
